### ISSUE TITLE
WinError 10060 Timeout Error when running semseg_fullyconv

### ISSUE THREAD

**Issue by CTO22** [dbuscombe-usgs/dl_tools#6]
Hi Daniel,

After 6 hours I finally got some data formatted correctly for semseg_fullyconv (misspelled name in my file RGB <> RBG)!  And when running the semantic segmentation approach I have reached about 11% and 13% complete during the "Creating the model" step then get the above timeout error.  Is this just something with the URL involved with the vgg grab? Or is there a timeout that can be increased?    Also when this fails I apparently need to delete the vgg.zip and vgg folder that get created, that sound correct?

Many thanks

**Comment by dbuscombe-usgs** [comment]
@CTO22 Hmm not sure, can you provide a screenshot of the error message. No, you don't need to delete the vgg.zip and vgg folder that get created. In fact if you didn't, the model training will be shorter because it won't have to download the model again.

**Comment by danhamill** [comment]
Here’s my two cents.  

@CTO22 This sounds like your internet connection timed out during the download.  I had the same problem and a second try solved this problem.

**Comment by CTO22** [comment]
It took about 10 tries, but eventually I had a stable enough connection to complete the download and the tool ran fine after that.  However, after each failed attempt I needed to delete the vgg zip file because it was only partially downloaded and I got a BadZip error if I didn't delete it...I haven't run the code again since the successful download, but I think it all should be good now.  Thank you Dans.
